src/Pages/MyBooks/Components/BookShelfList.js

```diff
@@ -14,20 +19,27 @@ export function BookShelfList({ bookShelfListCard, handleClickBookShelfList }) {
           <EditBtn
             onClick={() => {
               setModify(item.id);
+              setModifyToggle(!modifyToggle);
             }}
           >
             <HiOutlineDotsVertical />
           </EditBtn>
-          <Modify active={item.id === modify}>
+          <Modify active={item.id === modify && modifyToggle === true}>
```

Comment:
Boolean은 존재 자체로 값을 표현할 수 있습니다!
```suggestion
          <Modify active={item.id === modify && modifyToggle}>
```

Reply:
✔ 수정했습니다!